pkg/controller/scyllacluster/resource.go

```diff
@@ -625,6 +640,91 @@ func MakePodDisruptionBudget(c *scyllav1.ScyllaCluster) *v1beta1.PodDisruptionBu
 	}
 }
 
+func MakeIngresses(c *scyllav1.ScyllaCluster, services map[string]*corev1.Service) []*networkingv1.Ingress {
+	// Don't create Ingresses if options aren't provided.
+	if c.Spec.ExposeOptions == nil || c.Spec.ExposeOptions.Ingress == nil {
+		return nil
+	}
+
+	var ingresses []*networkingv1.Ingress
+
+	pathPrefix := networkingv1.PathTypePrefix
+	for _, service := range services {
+		var hosts []string
+		labels := naming.ClusterLabels(c)
```

Comment:
is this only being written to?

Reply:
I don't get what you mean, could you elaborate?